src/common/tusb_common.h

```diff
@@ -58,6 +58,9 @@
 // Generate a mask with bit from high (31) to low (0) set, e.g TU_GENMASK(3, 0) = 0b1111
 #define TU_GENMASK(h, l)      ( (UINT32_MAX << (l)) & (UINT32_MAX >> (31 - (h))) )
 
+// DCache padding for variable to occupy full cache line
+#define TUD_DCACHE_PADDING    uint8_t TU_XSTRCAT(dcache_padding_, _TU_COUNTER_)[CFG_TUD_MEM_DCACHE_ENABLE ? CFG_TUD_MEM_DCACHE_LINE_SIZE : 1]
```

Comment:
Can it be easier to read, when it is something like this? 
```suggestion
#if CFG_TUD_MEM_DCACHE_ENABLE 
#define TUD_DCACHE_PADDING    uint8_t TU_XSTRCAT(dcache_padding_, _TU_COUNTER_) [CFG_TUD_MEM_DCACHE_LINE_SIZE]
#else 
#define TUD_DCACHE_PADDING    
#endif
```

Because I can't get why we need to add member as `uint8_t dcache_padding_N[1];` when we don't need padding at all (no `CFG_TUD_MEM_DCACHE_ENABLE` configured).

Reply:
padding[1] does not have any impact at all since it is smallest size, it is irrelavant in a union.